lib/cocaine/command_line/runners/posix_runner.rb

```diff
@@ -42,6 +41,14 @@ def waitpid(pid)
         Process.waitpid(pid)
       end
 
+      def self.posix_spawn_gem_available?
+        require 'posix/spawn'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.